backend/lib/azimutt_web/admin/event/show.html.heex

```diff
@@ -0,0 +1,3 @@
+<div class="mx-auto px-12">
+    <h1 class="text-slate-100">Event #<%= assigns.event.id %></h1>
```

Comment:
Not sure this view is useful, I have no idea what to put inside except event fields (which are already visible on list)

Reply:
Data field of event

Reply:
Ok, why not.